### PR TITLE
Update recv_sd.ino

### PR DESCRIPTION
**Pull request by majorChange**
Fixes:
- replaced hard coded "4" with "SPI_CS_SD" to make the example more universal
- added wait for serial port to connect; needed for native USB port only;

### CODE REVIEW

File: examples/recv_sd/recv_sd.ino [hunk 1]
```diff
@@ -31,6 +31,9 @@ char str[20];
 void setup()
 {
     SERIAL.begin(115200);
+    while (!Serial) {
```
Comment (Pillar1989) [inline examples/recv_sd/recv_sd.ino]: Should be 
```
while (!SERIAL) {
```

Comment (majorChange) [reply examples/recv_sd/recv_sd.ino]: Ok. But where is the difference to what i wrote?
Please have a look at the example code [here](https://www.arduino.cc/reference/en/language/functions/communication/serial/ifserial/)

Comment (Pillar1989) [reply examples/recv_sd/recv_sd.ino]:
```
/*SAMD core*/
#ifdef ARDUINO_SAMD_VARIANT_COMPLIANCE
  #define SERIAL SerialUSB
#else
  #define SERIAL Serial
#endif
```
SERIAL means can be serial console, different platforms are not the same

Comment (majorChange) [reply examples/recv_sd/recv_sd.ino]: Now i see it. You are right. But the programm should wait for the serial console to be available. (Example: SAMD-variant right after uploading the code). So this line should stay `while (!Serial) {` but the line above is supposed to be `Serial.begin(115200);`

**Comment by Pillar1989** [comment]
Then I will close this PR and submit it to me after you update it

**Pull request closed without merging**